var/spack/repos/builtin/packages/libxrender/package.py

```diff
@@ -20,3 +20,12 @@ class Libxrender(AutotoolsPackage, XorgPackage):
     depends_on('renderproto@0.9:')
     depends_on('pkgconfig', type='build')
     depends_on('util-macros', type='build')
+
+    @property
+    def libs(self):
+        for dir in ['lib64', 'lib']:
+            libs = find_libraries('libXrender', join_path(self.prefix, dir),
+                                  shared=True, recursive=False)
```

Comment:
(comment, no change needed) Normally the default definition of `libs` should be able to find libraries when their names match the name of the package, but I'm guessing that doesn't work here because of the capitalization (i.e. Spack looks for `libxrender` but not `libXrender`). We should consider making that more flexible as it would remove the need for these definitions.

Reply:
Right now we're just using the glob library, which won't support ignoring case. We could use re instead, but it will be slower.

Reply:
What about a simple way to override the library name used for searching instead (e.g. adding a new property `library_name` to `Package`)?
Based on the docs for `glob` and `fnmatch` a case-insensitive `glob` should not be that much slower (only by how much `re` is slower for case insensitive matches and  internal optimizations of `fnmatch.filter` vs an explicit loop/comprehension).

Reply:
> What about a simple way to override the library name used for searching instead (e.g. adding a new property library_name to Package)?

That would work, although I'm inclined to say it would be preferable to allow for case-insensitive matching (I think this could be done without too much additional cost).